cfme/tests/automate/test_automate_manual.py

```diff
@@ -1025,3 +1025,120 @@ def test_automate_task_schedule():
         1713072
     """
     pass
+
+
+@pytest.mark.tier(2)
+@pytest.mark.meta(coverage=[1693362])
+def test_redhat_domain_sync_after_upgrade():
+    """
+    Polarion:
+        assignee: ghubale
+        initialEstimate: 1/8h
+        caseposneg: positive
+        casecomponent: Automate
+        testSteps:
+            1. Either dump database of appliance with version X to appliance with version Y
+               or upgrade the appliance
+            2. grep 'domain version on disk differs from db version' /var/www/miq/vmdb/log/evm.log
+            3. Check last_startup.txt file
+        expectedResults:
+            1.
+            2. You should find this string in logs: RedHat domain version on disk differs from db
+               version
+            3. You should find this string in file: RedHat domain version on disk differs from db
+               version
+
+    Bugzilla:
+        1693362
+    """
+    pass
+
+
+@pytest.mark.tier(2)
+@pytest.mark.meta(coverage=[1678135])
+def test_variable_pass_successive_playbook():
+    """
+    Polarion:
+        assignee: ghubale
+        initialEstimate: 1/8h
+        caseposneg: positive
+        casecomponent: Automate
+        setup:
+            1. Enable embedded ansible role
+        testSteps:
+            1. Add Ansible repo called billy -
+               https://github.com/billfitzgerald0120/ansible_playbooks
```

Comment:
@ganeshhubale can we use https://github.com/ManageIQ/integration_tests_playbooks instead of Bill's repo? That way we have authority/ability to maintain our own stuff. Dependency other external entity is reduced.

Reply:
Hey @kedark3 
Thanks this will be the better way. I have raised PR for adding playbooks.
https://github.com/ManageIQ/integration_tests_playbooks/pull/3